.dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.10.0
```

```suggestion
FROM python:3.10.5-slim-buster
```
latest stable py version